libensemble/tests/functionality_tests/test_executor_forces_tutorial_2.py

```diff
@@ -0,0 +1,72 @@
+import os
+import sys
+
+import numpy as np
+from forces_simf import run_forces  # Sim func from current dir
+
+from libensemble import Ensemble, logger
+from libensemble.alloc_funcs.start_only_persistent import only_persistent_gens as alloc_f
+from libensemble.executors import MPIExecutor
+from libensemble.gen_funcs.persistent_sampling import persistent_uniform as gen_f
+from libensemble.specs import AllocSpecs, ExitCriteria, GenSpecs, LibeSpecs, SimSpecs
+
+logger.set_level("DEBUG")
+
+if __name__ == "__main__":
+    # Initialize MPI Executor
+    exctr = MPIExecutor(custom_info={"mpi_runner": "openmpi"})
```

Comment:
Are we somehow running this only in the openmpi job?

Reply:
This came from copying/pasting the content from `docs/tutorials/executor_forces_tutorial.rst` to this test file. (see line 200 of the `develop` version of `docs/tutorials/executor_forces_tutorial.rst`)

This test is being invoked on Github actions with local comms, 4 workers. But (in general) openmpi is not available in the GH testing environment by default.